components/multiselect/multiselect_list.jsx

```diff
@@ -156,10 +160,25 @@ export default class MultiSelectList extends React.Component {
 
         const optionControls = options.map((o, i) => renderer(o, this.state.selected === i, this.props.onAdd));
 
+        if (this.props.ariaLabelRenderer) {
+            renderer = this.props.ariaLabelRenderer;
+        } else {
+            renderer = this.defaultAriaLabelRenderer;
+        }
+        const selectedOption = options.filter((o, i) => this.state.selected === i);
```

Comment:
Isn't the second argument of a filter just the index? Couldn't this just be 
```suggestion
        const selectedOption = options[this.state.selected];
```

Although if I'm wrong in that, we could replace the `filter` with a `find` to only get the first item that matches